link and text updates

edgeworthe opened the pull request:
Primarily updating the about page, but also temporarily removing
the links to nominate and jobs pages.

Commit:
remove nominate link

Currently the page isn't working on the static version
of the site, so let's remove for now in an
easily-revertable fashion since we have plans to get
it working at some point.

Commit:
remove jobs link

Currently isn't available, so remove the link until we're
ready to launch this feature.

Commit:
various text updates

Reworking the language to better fit blackswho.engineer;
probably will need further updates, but this feels like
a decent start.

edgeworthe commented on src/components/nav/index.js:
```diff
@@ -29,12 +29,6 @@ const Nav = props => {
-        </Link>
-        <Link to="/jobs" className={styles.link}>
-          Jobs
-        </Link>
```
I did these as two separate commits to make it easier to revert either if we add these pages back in

edgeworthe commented on src/pages/about.js:
```diff
@@ -21,49 +21,38 @@ class App extends React.Component {
-            to women who have never given a talk before.
+            Ensure that Black folks have prominent speaking slots.
+            Are Black women equally represented? Consider reaching 
+            out to Black folks who have never given a talk before.
```
I feel like I might be overusing "Black folks"—I'm _very_ open to suggestions for alternatives to decrease repetitiveness

TheBrotherFromASouthernMother replied:
Black folks sounds great to me, doesn't feel like it's being overused!

edgeworthe commented on src/pages/about.js:
```diff
@@ -152,16 +163,21 @@ class App extends React.Component {
+            by open-sourcing their code!
           </p>
           <p>
-            ✌️{" "}
```
doesn't show up very well in my web browser, but this is a ✌️ — I did consider leaving it in but adding the skin-tone (i.e. ✌🏿), so let me know if that feels better

TheBrotherFromASouthernMother replied:
I feel like this alright to remove. I don't think it adds significant value to the page

TheBrotherFromASouthernMother commented on src/pages/about.js:
```diff
@@ -21,49 +21,38 @@ class App extends React.Component {
             Blacks Who Engineer is a Twitter directory of
             accomplished Black folks in the
-            design industry. It aims to help people find notable and relevant
+            software industry. It aims to help people find notable and relevant
```
Regarding our Slack conversation about possibly expanding this directory to include other types of engineers, I wonder if it would make sense to change this from `software industry` to `tech industry`?

edgeworthe replied:
Sounds good, updated everywhere that we use this sentence, too

Commit:
Revert "remove nominate link"

This reverts commit 2ceb5b57b7fc249ee98a1f0f5d4e09f71e79c4c8.

I had hoped to do a clean revert, but the other link removal
after this caused a conflict.  I probably should've done
the nominate removal last since I knew a reversion would be
likely.

 Conflicts:
	src/components/nav/index.js

Commit:
s/software industry/tech industry/g

We hope to be more inclusive of engineering overall, so
replacing "software" with "tech".

That tagline gets used often enough that I wonder if it
should be a global variable, though.

TheBrotherFromASouthernMother commented:
LGTM!
